examples/tutorials/asr_inference_with_ctc_decoder_tutorial.py

```diff
@@ -222,28 +224,212 @@ def forward(self, emission: torch.Tensor) -> str:
 #
 # Now that we have the data, acoustic model, and decoder, we can perform
 # inference. Recall the transcript corresponding to the waveform is
-# ``"when it was the seven hundred and eighteenth night"``
+# ``"i really was very much afraid of showing him how much shocked i was at some parts of what he said"``
 #
 
+actual_transcript = "i really was very much afraid of showing him how much shocked i was at some parts of what he said"
 emission, _ = acoustic_model(waveform)
 
+
+######################################################################
+# The greedy decoder give the following result.
+#
+
+greedy_result = greedy_decoder(emission[0])
+greedy_transcript = greedy_result.replace("|", " ").lower().strip()
+greedy_wer = torchaudio.functional.edit_distance(actual_transcript.split(), greedy_transcript.split()) / len(
+    actual_transcript.split()
+)
+
+print(f"Transcript: {greedy_transcript}")
+print(f"WER: {greedy_wer}")
+
+
 ######################################################################
 # Using the beam search decoder:
+#
 
 beam_search_result = beam_search_decoder(emission)
 beam_search_transcript = " ".join(beam_search_result[0][0].words).lower().strip()
-print(beam_search_transcript)
+beam_search_wer = torchaudio.functional.edit_distance(actual_transcript.split(), beam_search_result[0][0].words) / len(
+    actual_transcript.split()
+)
+
+print(f"Transcript: {beam_search_transcript}")
+print(f"WER: {beam_search_wer}")
+
 
 ######################################################################
-# Using the greedy decoder:
+# We see that the transcript with the lexicon-constrained beam search
+# decoder produces a more accurate result consisting of real words, while
+# the greedy decoder can predict incorrectly spelled words like “affrayd”
+# and “shoktd”.
+#
 
-greedy_result = greedy_decoder(emission[0])
-greedy_transcript = greedy_result.replace("|", " ").lower().strip()
-print(greedy_transcript)
+
+######################################################################
+# Beam Search Decoder Parameters
+# ------------------------------
```

Comment:
not the scope of this PR but can we show how scores are computed? 

Like the one from [this](https://docs.google.com/presentation/d/1y9IHcy8F8rywgoTyvQbcsnlzzMbCSbPT2rW58pKGxms/edit) or cite a publication if there is as the linked one is more on the side of the implementation detail.

Reply:
sounds good, will leave this as a followup